### PR TITLE
Set default image

### PR DESCRIPTION
**Pull request by SarahhhC**
When sign in we replace null profile as default image!

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -386,48 +386,50 @@ function crawlByAccessTokenFacebook(accessToken) {
...
+
+        return user.save();
       } else {
         throw new Error(userCallback.ERR_MONGOOSE);
       }
     })
-    .then(keyData => {
-      if (keyData) {
-        if (req.body.image == null) {
+    .then(updatedUser => {
```
Comment (MechanicKim) [inline src/server/controllers/users.controller.js]: Refer this link ^^
https://github.com/airbnb/javascript#arrow-functions

File: src/server/controllers/users.controller.js [hunk 2]
```diff
@@ -386,48 +386,50 @@ function crawlByAccessTokenFacebook(accessToken) {
...
+        user.name = req.body.name;
+        user.email = req.body.email;
+        user.languages = req.body.languages;
+        user.location = req.body.location;
+        user.about = req.body.about;
+        user.education = req.body.education;
+        user.experience = req.body.experience;
+
+        return user.save();
       } else {
```
Comment (ochanje210) [inline src/server/controllers/users.controller.js]: isn't this unnecessary else statement

File: src/server/controllers/users.controller.js (hunk 1 above)
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: Thanks!

File: src/server/controllers/users.controller.js (hunk 2 above)
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: Don't need to use it?

**Comment by KyuaKwon** [comment]
LGTM

**Comment by KyuaKwon** [comment]
:shipit:

**Pull request merged**